plugins/poll/config/locales/client.en.yml

```
@@ -161,6 +163,8 @@ en:
           step: Step
         poll_public:
           label: Show who voted
+        poll_dynamic:
+          label: Enable dynamic mode (allow editing options after posting)
```

```suggestion
          label: Allow editing options after posting (dynamic poll)
```

I think the second half is more relevant to people than the "dynamic" wording